Sources/KSCrashRecording/Monitors/KSCrashMonitor_CPPException.cpp

```diff
@@ -172,7 +176,8 @@ catch(TYPE value)\
         ksmc_getContextForThread(ksthread_self(), machineContext, true);
 
         KSLOG_DEBUG("Filling out context.");
-        crashContext->crashType = KSCrashMonitorTypeCPPException;
+        crashContext->monitorName = g_monitorName;
+        crashContext->monitorProperties = g_monitorProperties;
```

Comment:
I like the cases when you have `name` and `properties` forward-declared instead of using constants. Could you please make this consistent?